source/code/include/playfab/PlayFabPlatformUtils.h

```diff
@@ -85,13 +85,13 @@ namespace PlayFab
     }
 
     // Get a tick count that represents now in milliseconds (not useful for absolute time)
-    inline long GetMilliTicks()
+    inline Int64 GetMilliTicks()
     {
 #if defined(PLAYFAB_PLATFORM_SWITCH)
         static_assert("You must request the Nintendo specific XPlat SDK from PlayFab support.");
 #else
         auto msClock = std::chrono::time_point_cast<std::chrono::milliseconds>(Clock::now());
-        return static_cast<long>(msClock.time_since_epoch().count());
+        return static_cast<Int64>(msClock.time_since_epoch().count());
```

Comment:
do you get some kind of compiler warning if you drop the static_cast? I would expect a long to implicitly cast to int64 just fine

Reply:
getting rid of casts is helpful because casts mask helpful warning messages (as discussed out of band)

Reply:
When it was "long long" to "long" yes, but now that it's Int64, no.
Thanks!

---
In reply to: [357854735](https://github.com/PlayFab/XPlatCoreTemplate/pull/65#discussion_r357854735) [](ancestors = 357854735)